korge/src/commonMain/kotlin/com/soywiz/korge/ui/UICheckBox.kt

```diff
@@ -0,0 +1,101 @@
+package com.soywiz.korge.ui
+
+import com.soywiz.korge.html.*
+import com.soywiz.korge.input.*
+import com.soywiz.korge.view.*
+import com.soywiz.korim.color.*
+import com.soywiz.korio.async.*
+import com.soywiz.korma.geom.*
+
+inline fun Container.uiCheckBox(
+	width: Number = 120.0,
+	height: Number = 32.0,
+	checked: Boolean = false,
+	text: String = "CheckBox",
+	textFont: Html.FontFace = defaultUIFont,
+	skin: UISkin = defaultUISkin,
+	checkedSkin: UISkin? = null,
```

Comment:
I would totally drop the X text and would include a texture slice for the checkbox in its states as part of the UISkin itself

Reply:
Still we can do this later, the parameter

Reply:
I would drop the X text too, but the DefaultUISkin doesn't have any alternative for the X sign, so the checked state of the default implementation of uiCheckBox (with defaultUISkin) will not be displayed correctly. So in order to drop it, we need to provide a correct default skin.